app/controllers/api/v1/items/random_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::Items::RandomController < ApplicationController
+
+  def show
+    render json: Item.order("RANDOM()").first
```

Comment:
i'd suggest pulling this query out into a model method so we can potentially reuse it. this is a lot of logic for the controller 👍